INSTALL.md

```diff
@@ -0,0 +1,68 @@
+# Install Tomateos
+### Required dependencies
+
+Required dependencies trough Packet-Managers/Manual installation:
+* Ruby >= 2.3.3
+* Ruby-Gems
+* MongoDB (More installation instructions at: https://docs.mongodb.com/manual/administration/install-on-linux/ for your desired distribution)
+
+If you wan't to install it with RVM, you will need RVM of course too.
+##### Install ruby with RVM(works with nearly all distros)
+
+Install RVM:
+```
+su
+$PackageManager install rvm
+```
+Replace $PackageManager with your desired manager ;)
+
+
+With RVM now install ruby in version 2.3.3(No root required):
+```sh
+rvm use 2.3.3 --install --binary --fuzzy
+```
+Output will be most likely something like: 
+```
+ruby-2.3.3 is not installed - installing.
+[...]
+Using ~/.rvm/gems/ruby-2.3.3
+```
+Now you have only to install mongodb yourself:
+```
+su
+yum install mongodb
+apt-get install mongodb
+```
+
+Fire up MongoDB:
+`sudo service mongodb start`
+
+And proceed with the `gem` install.
+##### Installation trough distributions Packet Manager:
```

Comment:
Alright, seem's legit. Mostly because of the APT-things, it is a little bit confusing.